README.md

```diff
@@ -25,14 +25,20 @@ In this challenge, create a web API around the following resources: `Projects` a
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] Mention two parts of Express that you learned about this week.
+- {* it is written in JavaScript and used within Node.js
+   * we can create and use our own localhost http and route it to different urls that we create as well. }
 
 - [ ] Describe Middleware?
+- { a fuction that allows the parts of the app to communicate and save data to allow to other parts of the app. It is a 'middleman' that tells the app whether or not this data will be stopped, changed or continue through the chain.}
 
 - [ ] Describe a Resource?
+- { a place in which the api is going to ie. id is a resource.}
 
 - [ ] What can the API return to help clients know if a request was successful?
+- { it can return a status code in the 200s }
 
 - [ ] How can we partition our application into sub-applications?
+- { By using  Express middleware }
```

Comment:
technically that is true, but I think the answer they are looking for is Express Router. The express router is just big ole middleware.